src/test/ui/let-else/let-else-run-pass.rs

```diff
@@ -11,7 +11,7 @@ fn main() {
     }
     // ref binding to non-copy value and or-pattern
     let (MyEnum::A(ref x) | MyEnum::B { f: ref x }) = (MyEnum::B { f: String::new() }) else {
-        panic!();
+        panic!("Shouldn't have matched enum");
```

Comment:
What prompted the changes?

Reply:
I couldn't tell which one was matching the panic, I forget why line numbers weren't sufficient